src/components/DisplayWeb3.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react'
+import ReactDOM from 'react-dom'
+
+import { Web3Api } from '../types'
+
+interface DisplayWeb3Props {
+    web3Api: Web3Api
+}
+
+interface ClientVersion {
+    id?: number
+    jsonrpc?: string
+    result: string
+}
+
+export const DisplayWeb3: React.FC<DisplayWeb3Props> = ({ web3Api }) => {
+    const [account, setAccount] = useState<string>()
+
+    const [balance, setBalance] = useState<string>()
+
+    const [networkName, setNetworkName] = useState<string>()
+
+    const [clientVersion, setClientVersion] = useState<ClientVersion>()
+
+    useEffect(() => {
+        const setter = async (): Promise<void> => {
+            const accountsProm = web3Api.getAccounts()
```

Comment:
I think it's better not to shorten the word promise :)

![image](https://user-images.githubusercontent.com/2352112/63793025-11103000-c8ff-11e9-8156-a1a04069afc1.png)